prac_05/hex_colours.py

```diff
@@ -0,0 +1,16 @@
+"""
+CP1404 Practical
+Hexadecimal colour code program
+"""
+
+COLOUR_TO_CODE = {"Raspberry": "#e30b5d", "Red1": "#ff0000", "Blue1": "#0000ff", "Green1": "#00ff00",
+                  "Gunmetal": "#2a3439", "MediumPurple": "#9370db", "Ash Grey": "#b2beb5",
+                  "Baby Pink": "#f4c2c2", "Banana Yellow": "#ffe135", "Burnt Orange": "#cc5500"}
+
+name = input("Enter colour name: ").title()
```

Comment:
While name does work as a variable name here in the future it may be beneficial to make it more specific

Reply:
You're right, thankyou for pointing that out!